docs/guides/launch.md

```diff
@@ -57,287 +57,83 @@ access it at `http://127.0.0.1:3000/` under credentials `admin/admin`.
 ## (Re)deploy db and contracts
 
 ```
-zk contract redeploy
+zkstack dev contracts
```

Comment:
you don't need to build them manually

Reply:
Addressed in https://github.com/matter-labs/zksync-era/pull/3134/commits/3dfc55eb2a4b489afd1eef24c1eff82721662776